msgraph/models.go

```diff
@@ -44,16 +41,14 @@ type AccessPackageAssignmentPolicy struct {
 }
 
 type AccessPackageCatalog struct {
-	ID                  *string                    `json:"id,omitempty"`
-	CatalogStatus       AccessPackageCatalogStatus `json:"catalogStatus,omitempty"`
-	CatalogType         AccessPackageCatalogType   `json:"catalogType,omitempty"`
-	CreatedBy           *string                    `json:"createdBy,omitempty"`
-	CreatedDateTime     *time.Time                 `json:"createdDateTime,omitempty"`
-	Description         *string                    `json:"description,omitempty"`
-	DisplayName         *string                    `json:"displayName,omitempty"`
-	IsExternallyVisible *bool                      `json:"isExternallyVisible,omitempty"`
-	ModifiedBy          *string                    `json:"modifiedBy,omitempty"`
-	ModifiedDateTime    *time.Time                 `json:"modifiedDateTime,omitempty"`
+	ID                  *string                   `json:"id,omitempty"`
+	State               AccessPackageCatalogState `json:"state,omitempty"`
```

Comment:
As above, could we keep both fields for now in order to maintain compatibility with the beta API? I'm assuming the beta docs are correct and the field is still called `catalogStatus` in the beta model?